src/common/gui/MSEGEditor.cpp

```diff
@@ -1118,13 +1131,12 @@ struct MSEGCanvas : public CControl, public Surge::UI::SkinConsumingComponent, p
                if (h.useDrawRect)
                   r = h.drawRect;
 
-               int cx = r.getCenter().x;
+               auto cx = r.left;
 
-               /*dc->setLineWidth(1);
-               dc->setFrameColor(kRedCColor);
-               dc->drawRect(r, kDrawStroked);*/
+               if (h.zoneSubType == hotzone::LOOP_END)
+                  cx = r.right;
 
-               if (cx >= drawArea.left && cx <= drawArea.right)
+               if (cx >= drawArea.left || cx <= drawArea.right)
```

Comment:
This or to and swap is i think a mistake no?

Reply:
Yep!